_specs/ecip-1076.md

```diff
@@ -1,58 +1,39 @@
 ---
 lang: en
 ecip: 1076
-title: Mining Algorithm Change Process
+title: Mining Algorithm Signaling Process
 author: Talha Cross (@soc1c)
 discussions-to: https://github.com/ethereumclassic/ECIPs/issues/174
-status: Draft
+status: Active
```

Comment:
This is invalid. Signaling is a hard fork process, and thus this should follow "Last Call" -> "Accepted" -> "Final" route, instead of "Active".

Reply:
It's not a hard fork process. It's non-binding signaling.

Reply:
@soc1c Still, there's no justification to move this to "Active" status. I believe that's just wrong.

Reply:
Quote ECIP-1000:

> _"Process ECIPs may also be moved to a status of “Active” instead"_

This clearly is a process ECIP.

Reply:
@soc1c I would put this in the same category as specifications like ERC-20. I don't believe they can be moved to "Active" status, thus the same for this ECBP.